blobstore-fs/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "blobstore-fs"
```

Comment:
There's a naming convention we typically follow when building providers so that they all appear similarly on crates. The name for this package under that convention would be `wasmcloud-provider-blobstore-fs` (the S3 capability provider for this contract is called `wasmcloud-provider-blobstore-s3`.

Reply:
Will be fixed in next commit.